repositories/task_repository.py

```diff
@@ -19,3 +19,25 @@ def create(self, task: Task) -> Task:
         self.__db_context.commit()
         self.__db_context.refresh(task)
         return task
+
+    def update(self, task: Task) -> ServiceResponse:
```

Comment:
Почему метод репозитория возвращает service response, а не доменную сущность? 

Мне кажется, тут лучше было бы сделать на ретерн 
```suggestion
    def update(self, task: Task) -> List[Task]:
```

А сам ответ нас волнует только непосредственно в самой ручке.
Вдруг эта логика должна будет встраиваться, например, в логику отработки эвентов из очереди? Тогда ServiceResponse со статусом кода 200 не подойдет.

Reply:
Соре, не List[Task], а просто Task